runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -9806,6 +9806,36 @@ static TR::Register *inlineSinglePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT_FATAL(node->getNumChildren() == 3, "There should be exactly 3 children in the node of inlineFPTrg1Src3");
+
+   TR::DataType type = node->getDataType();
+   TR_ASSERT_FATAL(type == TR::Float || type == TR::Double, "Node of inlineFPTrg1Src3 should be either TR::Float or TR::Double");
```

Comment:
Perhaps you could output the node address as part of the message.

Reply:
Fixed in 05e6a71